src/content/1.1/code/agda/snippet05.agda

```diff
@@ -1,2 +1,2 @@
 id : ∀ {A} → A → A
-id a = a
\ No newline at end of file
+id a = a
```

Comment:
I don't know what changed here.

Reply:
Here and in e.g. in other places like `snippet03.agda` you are adding newline at the end of the file. Technically this is correct, but for some case snippets does not contains them.

Normally people insist on having newline at the end of the every text file. Otherwise it is not technically text file. That is why GH shows red circle below old version. To note newline is missing.

We can go for newlines and if we have PDF version working test if there is any reason to remove them. It should be easy to automate.